src/apps_plugin.c

```diff
@@ -245,6 +246,9 @@ long get_system_cpus(void) {
 
 long get_system_pid_max(void) {
 	procfile *ff = NULL;
+
+	// FIXME: Is this a typical maximum system pid count, or just for Linux?
+	//        Is it defined on POSIX.1 ABI X/Open and/or BSD specs?
 	long mpid = 32768;
```

Comment:
what exactly should one read from there?

i have system where this value is exceeded several times:

```
$ cat /proc/sys/kernel/pid_max
262136
```

http://stackoverflow.com/questions/6294133/maximum-pid-in-linux

from programming perspective you should use `pid_t` type